examples/example_basic.lua

```diff
@@ -16,7 +16,12 @@ local function TestOneInput(buf)
 end
 
 local args = {
-    only_ascii = 1,
-    print_pcs = 1,
+    "-only_ascii=1",
+    "-print_pcs=1",
 }
-luzer.Fuzz(TestOneInput, nil, args)
+
+for i = 1, #args do
+    arg[#arg + 1] = args[i]
+end
```

Comment:
I don't like this. User should not rewrite `arg` table in scripts.

Lua has an `arg` variable with all specified command-line options:

```
$ cat script.lua 
require('log').info(arg)
$ tarantool script.lua -a=1 -b=2
{"0":"script.lua","1":"-a=1","2":"-b=2","-1":"\/usr\/bin\/tarantool"}
$ 
```
Seems we need to convert a map to array in an implementation of a `Fuzz` function and pass to `TestFuzzOneInput`.

Reply:
Not relevant

Reply:
Fixed it